compiler/one-cmds/one_utils.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import configparser
+import os
+import subprocess
+import sys
+
+
+def _add_default_arg(parser):
+    # version
+    parser.add_argument(
+        '-v',
+        '--version',
+        action='store_true',
+        help='show program\'s version number and exit')
+
+    # configuration file
+    parser.add_argument('-C', '--config', type=str, help='run with configuation file')
+    # section name that you want to run in configuration file
+    parser.add_argument('-S', '--section', type=str, help=argparse.SUPPRESS)
```

Comment:
do we need this? as we discussed off-line, we could remove this for the first version.

Reply:
@seanshpark if `argparse.SUPPRESS` is given to `help` kwarg, user can't see its option. It is internal or private option for implementation convenience and scalability.

Reply:
Can user add this in argument and override it? Users can still access the source code.

Reply:
Yes. Umm.. I think there is no demerit even if user finds out this option from source code. If you worries about misbehave like randomly user gives `-S`, we can print warning at this time.

Reply:
OK, got it. I think no need to add warning message. Using -`S` is possible but it's up to the user to use unofficial arguments and get misbehaviour.